backend/source/types/SourceType.py

```diff
@@ -0,0 +1,15 @@
+from graphene import ResolveInfo
+from django.db.models import QuerySet
+from graphene_django import DjangoObjectType
+from source.models import Source
+
+
+class SourceType(DjangoObjectType):
+    class Meta:
+        model = Source
+
+    @classmethod
+    def get_queryset(
```

Comment:
I usually define `get_queryset()` on every type. It is used to filter the queryset, most often based on the user making the request (obtainable via `info.context.user`). It's not necessary (and in fact `return queryset.all()` is the default implementation), but it is nice to be explicit about returning _all_ objects of a particular type.